tools/tests/test_lint.py

```
@@ -252,6 +252,90 @@ def test_ledger_row_bad_vocab_values_are_findings(tmp_path):
     assert any("disposition" in f and "vibes" in f for f in findings)
 
 
+def test_ledger_vocabularies_are_exactly_what_the_adr_states(tmp_path):
```

Ruff (0.16.1)

[warning] Unused function argument: `tmp_path`

(ARG001)